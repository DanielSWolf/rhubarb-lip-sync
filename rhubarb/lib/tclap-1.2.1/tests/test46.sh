#!/bin/sh

# success
../examples/test7 --help  > tmp.out 2>&1

if cmp -s tmp.out $srcdir/test46.out; then
	exit 0
else
	exit 1
fi

