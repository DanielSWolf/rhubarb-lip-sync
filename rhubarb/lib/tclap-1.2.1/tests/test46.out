
USAGE: 

   ThisIsAVeryLongProgramNameDesignedToTestSpacePrintWhichUsedToHaveProblem
                                        sWithLongProgramNamesIThinkItIsNowL
                                        ongEnough  [-l <int>] [-u <float
                                        lte 1>] [-b <float lte 1>] [-z
                                        <filename>] [-x <filename>] [-s
                                        <name string>] [-d] [-g <negative
                                        int>] [-f <negative int>] -n <homer
                                        |marge|bart|lisa|maggie> ...  [--]
                                        [--version] [-h] <1|2|3> ...


Where: 

   -l <int>,  --limit <int>
     Max number of alignments allowed

   -u <float lte 1>,  --upperBound <float lte 1>
     upper percentage bound

   -b <float lte 1>,  --lowerBound <float lte 1>
     lower percentage bound

   -z <filename>,  --filename2 <filename>
     Sequence 2 filename (FASTA format)

   -x <filename>,  --filename1 <filename>
     Sequence 1 filename (FASTA format)

   -s <name string>,  --scoring--Matrix <name string>
     Scoring Matrix name

   -d,  --isDna
     The input sequences are DNA

   -g <negative int>,  --gap-Extend <negative int>
     The cost for each extension of a gap

   -f <negative int>,  --gapCreate <negative int>
     The cost of creating a gap

   -n <homer|marge|bart|lisa|maggie>,  --name <homer|marge|bart|lisa
      |maggie>  (accepted multiple times)
     (required)  Name to print. This is a long, nonsensical message to test
     line wrapping.  Hopefully it works.

   --,  --ignore_rest
     Ignores the rest of the labeled arguments following this flag.

   --version
     Displays version information and exits.

   -h,  --help
     Displays usage information and exits.

   <1|2|3>  (accepted multiple times)
     Number of times to print


   Command description message. This is a long multi-line message meant to
   test line wrapping.  This is more text that doesn't really do anything
   besides take up lots of space that otherwise might be used for something
   real.  That should be enough, don't you think?

